Search/BingImageSearch/Quickstart/README.md

```diff
@@ -1,44 +1,43 @@
 
-Sample code for searching images using Bing Image Search, an Azure Cognitive Service.
-- Search images for "canadian rockies" then verify number of results and print out pivot suggestion and query expansion.
-- Search images for "studio ghibli", filtered for animated GIFs and wide aspect and print out insightsToken, thumbnail url and web url.
-- Search for trending images then print out categories and tiles.
-- Search images for "degas" and then search the image details of the first item in result list.
+This code sample compliments the [Bing Image Search Quickstart](https://docs.microsoft.com/azure/cognitive-services/bing-image-search/image-sdk-java-quickstart) available on docs.microsoft.com. 
 
+This Java application sends an image search query to the Bing Image Search API, parses the JSON response, and displays the URL of the first image returned, along with some other information.
 
-## Features
-
-This project framework provides examples for the **Bing Image Search SDK** for the [Image Search API](https://azure.microsoft.com/en-us/services/cognitive-services/)
+The [Bing Image Search API](https://azure.microsoft.com/services/cognitive-services/bing-image-search-api/) is an Azure Cognitive service, available from Microsoft.  
 
 ## Getting Started
 
 ### Prerequisites
 
-- A cognitive services API key with which to authenticate the SDK's calls. [Sign up here](https://azure.microsoft.com/en-us/services/cognitive-services/directory/) by navigating to the **Search** services and acquiring an API key. You can get a trial key for **free** which will expire after 30 days.
+- The latest version of the [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/index.html) (JDK)
 - Maven
 
-### Quickstart
+You must have a [Cognitive Services API account](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account) with access to the Bing Search APIs. If you don't have an Azure subscription, you can [create an account](https://azure.microsoft.com/try/cognitive-services/?api=bing-web-search-api) for free. Before continuing, You will need the access key provided after activating your free trial, or a paid subscription key from your Azure dashboard.
 
-To get these samples running locally, simply get the pre-requisites above, then:
+### Quickstart
 
+After getting the prerequisites above:
+1. View the quickstart available [here]([Quickstart: Search for images using the Bing Image Search SDK and Java](https://docs.microsoft.com/azure/cognitive-services/bing-image-search/image-sdk-java-quickstart))
 1. git clone https://github.com/Azure-Samples/cognitive-services-java-sdk-samples.git
-2. cd cognitive-services-java-sdk-samples/Search/BingImageSearch
-3. set env variable AZURE_BING_SAMPLES_API_KEY to your cognitive services API key.
+2. cd cognitive-services-java-sdk-samples/Search/BingImageSearch/Quickstart
+3. Copy your cognitive services API key into the program.
```

Comment:
For consistency purposes with the other Cognitive Services samples and simplicity can you please update the code to read the key from the environment variable? I would expect that most of the programs out there will use this technique rather than hard coding the key in their application.

Reply:
Thanks, most programs would likely read from the environment, but the scope of this code sample is to demonstrate the API in five minutes. I worry the target audience (which includes newer developers on up) may not be familiar with the process, or want to set the environment key for the example. 

If I included the option of setting it either way, would that help with consistency? Something like this:
```java
//uncomment the below line if you have set the ` AZURE_BING_SAMPLES_API_KEY` to your cognitive services API key.
//System.getenv("AZURE_BING_SAMPLES_API_KEY");
//otherwise uncomment this line and paste your subscription key in
//string subscriptionKey = "Your key here";
```

Reply:
+ @selvasingh 

The sample should be runnable without any changes to the code from the command line as long as the environment is properly set (in this case the env variable points to a valid subscription key). I'm not the one making these expectations up, I've been given the same feedback on the very first Java sample I wrote :-)

It will be great if you can change the code to simply read from the environment variable as default; you can add a preceding comment though to say something in line with what you had: "paste your subscription key here instead of reading from the environment variable".